### PR TITLE
Adding behat test for attribute, languages, deepl, workflow

### PR DESCRIPTION
**Pull request by wiewiurdp**

### CODE REVIEW

File: features/workflow.feature [hunk 1]
```diff
@@ -54,6 +278,230 @@ Feature: Workflow
...
+        }
+      """
+    When I request "/api/v1/EN/status/@workflow_status@" using HTTP PUT
+    Then validation error response is received
+
+  Scenario: Update status (without description)
+    Given current authentication token
+    Given the request body is:
+      """
+          {
```
Comment (BastekBielawski) [inline features/workflow.feature]: Check formatting

File: features/deepl.feature [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Feature: Deepl module
```
Comment (BastekBielawski) [inline features/deepl.feature]: We need to pay for those requests?

File: features/workflow.feature [hunk 3]
```diff
@@ -21,6 +21,230 @@ Feature: Workflow
...
+      }
+      """
+    When I request "/api/v1/EN/status" using HTTP POST
+    Then created response is received
+
+  Scenario: Create status (wrong language parameter)
+    Given current authentication token
+    Given the request body is:
+      """
+          {
```
Comment (BastekBielawski) [inline features/workflow.feature]: Check formatting

File: features/workflow.feature [hunk 4]
```diff
@@ -21,6 +21,230 @@ Feature: Workflow
...
+      """
+    When I request "/api/v1/EN/status" using HTTP POST
+    Then validation error response is received
+
+  Scenario: Create status (empty name)
+    Given current authentication token
+    Given the request body is:
+      """
+      {
+        "color": "#ff0000",
```
Comment (BastekBielawski) [inline features/workflow.feature]: Check color format "#f00". It should be valid

File: features/workflow.feature [hunk 5]
```diff
@@ -21,6 +21,230 @@ Feature: Workflow
...
+      }
+      """
+    When I request "/api/v1/EN/status" using HTTP POST
+    Then created response is received
+
+  Scenario: Create status (empty description)
+    Given current authentication token
+    Given the request body is:
+      """
+          {
```
Comment (BastekBielawski) [inline features/workflow.feature]: Check formatting

File: features/workflow.feature [hunk 6]
```diff
@@ -21,6 +21,230 @@ Feature: Workflow
...
+        }
+      """
+    When I request "/api/v1/EN/status" using HTTP POST
+    Then validation error response is received
+
+  Scenario: Create status (without description)
+    Given current authentication token
+    Given the request body is:
+      """
+          {
```
Comment (BastekBielawski) [inline features/workflow.feature]: Check formatting

File: features/workflow.feature [hunk 7]
```diff
@@ -21,6 +21,230 @@ Feature: Workflow
...
+        "color": "#ff0000",
+        "code": "ST @@random_md5@@",
+        "name": {
+          "PL": "PL",
+          "EN": "EN"
+        },
+        "description" :""
```
Comment (BastekBielawski) [inline features/workflow.feature]: Emty description should be ``{}``

File: features/workflow.feature [hunk 8]
```diff
@@ -21,6 +21,230 @@ Feature: Workflow
...
+    Then validation error response is received
+
+  Scenario: Create status (empty name)
+    Given current authentication token
+    Given the request body is:
+      """
+      {
+        "color": "#ff0000",
+        "code": "ST @@random_md5@@",
+        "name": "",
```
Comment (BastekBielawski) [inline features/workflow.feature]: Empty name should be ``{}``

File: features/workflow.feature [hunk 9]
```diff
@@ -54,6 +278,230 @@ Feature: Workflow
...
+      }
+      """
+    When I request "/api/v1/EN/status/@workflow_status@" using HTTP PUT
+    Then empty response is received
+
+  Scenario: Update status (wrong language parameter)
+    Given current authentication token
+    Given the request body is:
+      """
+          {
```
Comment (BastekBielawski) [inline features/workflow.feature]: Format

File: features/workflow.feature [hunk 10]
```diff
@@ -54,6 +278,230 @@ Feature: Workflow
...
+      }
+      """
+    When I request "/api/v1/EN/status/@workflow_status@" using HTTP PUT
+    Then created response is received
+
+  Scenario: Update status (empty description)
+    Given current authentication token
+    Given the request body is:
+      """
+          {
```
Comment (BastekBielawski) [inline features/workflow.feature]: Format

File: features/deepl.feature (hunk 2 above)
Comment (wiewiurdp) [reply features/deepl.feature]: Yes, 0,01 Euro for 500 characters and in test there is only word with 3 characters. I was aware of this and I thought it is not a problem, What do you think about that? We should comment Deepl tests?

Comment (BastekBielawski) [reply features/deepl.feature]: Yes, we should :) We need to find mock deepl client

**Pull request merged**